docs/source/common/loggers.rst

```diff
@@ -129,12 +133,16 @@ The :class:`~pytorch_lightning.loggers.NeptuneLogger` is available anywhere exce
 
     class MyModule(LightningModule):
         def any_lightning_module_function_or_hook(self):
-            some_img = fake_image()
-            self.logger.experiment.add_image("generated_images", some_img, 0)
+            # generic recipe for logging custom metadata
+            metadata = ...
+            self.logger.experiment["your/metadata/structure"].log(metadata)
```

Comment:
Mind adding a comment this API won't work with a collection of loggers.

Reply:
@Herudaio it's something for you.

Reply:
@tchaton @shnela @Herudaio comment added